avalanche/logging/wandb_logger.py

```diff
@@ -28,17 +27,19 @@ class WandBLogger(StrategyLogger):
     logged to a dedicated Weights & Biases project dashboard.
     """
 
-    def __init__(self, init_kwargs: dict, interactive: bool = False):
+    def __init__(self, project_name: str, run_name: str, params: dict):
         """
         Creates an instance of the `WandBLogger`.
 
-        :param init_kwargs: All arguments for wandb.init() function call.:
+        :param params: All arguments for wandb.init() function call.:
```

Comment:
If maintaining here the list of arguments is challenging, can you put a link to the W&B documentation for `init` call?

Reply:
@AndreaCossu if I understood correctly you want me to put a URL to the W&B init params docs in the docstrings?

Reply:
exactly :smile: